Cargo.toml

```diff
@@ -10,8 +10,6 @@ documentation = "http://ironframework.io/doc/router/index.html"
 keywords = ["iron", "web", "http", "routing", "router"]
 
 [dependencies]
+iron = "~0"
```

Comment:
I got it. If we set it `~0` the older version might be broken in future and users won't have chance to downgrade.